packages/react-testing/src/element.ts

```diff
@@ -252,3 +258,14 @@ function equalSubset(subset: object, full: object) {
     key => key in full && full[key] === subset[key],
   );
 }
+
+function getDescendants(element) {
+  const descendants: Element<unknown>[] = [];
+  // eslint-disable-next-line @typescript-eslint/prefer-for-of
```

Comment:
Could we actually use `for..of` here?

Reply:
I'm going to leave the for loop, the iterators are too much of a hit. Wonder if it would be faster if not compiled to es5 🤔 

for loop
<img width="1011" alt="Screen Shot 2021-04-01 at 2 31 13 PM" src="https://user-images.githubusercontent.com/3513040/113338429-19102d00-92f7-11eb-8da6-f7ff851f80e1.png">

for of
<img width="1011" alt="Screen Shot 2021-04-01 at 2 30 37 PM" src="https://user-images.githubusercontent.com/3513040/113338436-1b728700-92f7-11eb-913d-dadc8f460783.png">

Reply:
Oof.